sphinx_immaterial/__init__.py

```diff
@@ -271,6 +272,10 @@ def setup(app):
 
     app.add_builder(_get_html_builder(app.registry.builders["html"]), override=True)
     app.add_html_theme("sphinx_immaterial", os.path.abspath(os.path.dirname(__file__)))
+
+    # register our custom adminition directive
+    custom_admonition_setup(app)
```

Comment:
Could just have the user add `sphinx_immaterial.md_admonition` as an additional extension in the `conf.py` if they need this.

Alternatively, if you strongly prefer to load it by default you can call: `app.setup_extension('sphinx_immaterial.md_admonition')`

Reply:
I would prefer to have it loaded by default since it is meant to help people migrating from mkdocs-material (& it is documented as part of the theme's featured admonition directives). I can change the function call...

I should probably make a note in the docs about the ability to use it as independent extension in other themes.